src/features/chat/ChatItem.tsx

```diff
@@ -0,0 +1,57 @@
+import React /*, { CSSProperties }*/ from 'react'
+import cn from 'classnames'
+import { formatTime } from '../../common/utils/format'
+import { ChatMessageProps } from './ChatMessage'
+import { UserAvatar } from './components/UserAvatar'
+import styles from './Chat.module.css'
+
+export interface ChatItemProps {
```

Comment:
Please use type instead of interface.
And add T prepend => TChatItemProps

Reply:
changed